frontend/src/components/VirtualList/Config.ts

```diff
@@ -28,8 +29,17 @@ export const hasHealth = (r: RenderResource): r is SortResource => {
   return (r as SortResource).health !== undefined;
 };
 
-export const hasMissingSidecar = (r: SortResource): boolean => {
-  return !isIstioNamespace(r.namespace) && !r.istioSidecar && !isGateway(r.labels) && !isWaypoint(r.labels);
+export const hasMissingSidecar = (workload: Workload | WorkloadListItem | AppWorkload | AppListItem): boolean => {
+  return (
+    // TODO:
+    // Are we missing an ambient case here where ambient is enabled AND we are missing the ambient labels AND we are missing the sidecar?
+    // hasMissingSC && hasMissingA && serverConfig.ambientEnabled
```

Comment:
But, in this case, workload.isAmbient should be false

Reply:
Forgot about my last comment, in the case of Ambient, that check would be needed also: 
![image](https://github.com/user-attachments/assets/51a276fe-315e-4c98-a2c4-403f4662dd15)

Reply:
I think this has more to do with the new gateway label. Can you test again with the latest changes?